resources/lang/en/events/death/general.json

```
@@ -5863,14 +5896,8 @@
         "season": [
             "any"
         ],
-        "tags": [
-            "clan:kitten"
-        ],
-        "subtype": [
-            "war"
-        ],
         "weight": 20,
-        "event_text": "m_c was killed defending the kits from o_c_n warriors.",
+        "event_text": "o_c_n attacked the camp and m_c was killed defending the kits from enemy warriors.",
```

Hot take but I think this is less legible and more words. May I suggest...?

```suggestion
        "event_text": "m_c was killed defending the kits during a o_c_n raid.",
```